lib/client.js

```diff
@@ -192,9 +310,16 @@ var GiantSwarm = (function () {
      * Create a websocket connection to stream logs from instances
      */
     streamLogs: function(organizationName, instanceIds, messageCallback, createCallback, errorCallback) {
+      this._validateStringParameters({
+        organizationName: organizationName,
+      });
+      if (typeof(instanceIds) !== 'object') throw new this.UserException("Parameter 'instanceIds' must be an array");
```

Comment:
FYI this does not check that the parameter is really an array, because JS sucks. IN general for things like this I would suggest to user the `underscore` library.